scripts/minishiftci-presubmit-all-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
I made the same comment at  https://github.com/openshift/odo/pull/4401#discussion_r588263997

> This script looks almost the same as what we already have in https://github.com/openshift/odo/blob/master/scripts/openshiftci-e2e-4x-psi-tests.sh
> 
> We should reuse what we already have and not create duplication. If needed the existing script should be generalized or extracted into a function.

This makes it 3rd instance of almost the same code.

Reply:
I will consolidate into one as well.